bitnami/airflow/templates/role.yaml

```
@@ -0,0 +1,28 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ template "airflow.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "airflow.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "airflow.tplValue" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations:
+    {{- include "airflow.tplValue" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
+  {{- end }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+      - pods/log
+      - pods/exec
+      - services
+      - endpoints
+      - configmaps
+      - secrets
+    verbs:
+      - '*'
```

```suggestion
      - get
      - list
      - watch
      - create
      - delete
      - update
      - patch
```

For clarifications sake, please expand to a list the verbs. Also, I'm not sure if KubernetesPodOperator need permissions so broad to function. Could you confirm this?

---

@FraPazGal, Hmmm... Removed update/patch. Rest all is needed as far as I understood from reading airflow kubernete pod operator code [here](https://github.com/apache/airflow/tree/master/airflow/kubernetes).

I have removed `services/endpoints` from `resources` cause we run spark-on-kubernetes, but for that, we can specify a different service account if needed and that's the right way to do it not sharing airflow service account with spark driver.